tests/authentication/registration/test_verify.py

```diff
@@ -19,38 +18,53 @@ class UserEmailVerificationTest(BaseTest):
-        response = self.client.get(self.verify_url+"?token=" + jwt.encode(
-            {"email": self.new_user["email"]}, settings.SECRET_KEY,
-            algorithm="HS256").decode("utf-8")+"~6")
+        response = self.client.get(
+            self.verify_url+"?token={}".format(valid_token))
+        print('======', response.content)
         self.assertEqual(
-            response.status_code, status.HTTP_200_OK)
+            response.status_code, status.HTTP_302_FOUND)
 
     @patch('utils.tasks.send_email_notification.delay')
     def test_user_cant_activate_with_expired_token(self, mock_email):
         """Test should not activate an account when the link is expired."""
         mock_email.return_value = True
-        expired_token = jwt.encode({"email": self.user.email, "exp": datetime.utcnow() -
-                                    timedelta(hours=23)}, settings.SECRET_KEY).decode("utf-8")+"~"+str(self.user.id)
+        data = {"email": self.user.email,
+                "exp": datetime.utcnow() - timedelta(hours=23)}
+        expired_token = jwt.encode(data,
+                                   settings.SECRET_KEY)\
+            .decode("utf-8")+"~"+str(self.user.id)
         response = self.client.get(
-            self.verify_url+"?token="+expired_token, format="json")
+            self.verify_url+"?token={}".format(expired_token), format="json")
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
     def test_user_cant_not_activate_twice(self):
         """Test should not activate an account twice."""
         self.user.is_verified = True
         self.user.save()
-        response = self.client.get(self.verify_url+"?token="+jwt.encode(
-            {"email": self.user.email}, settings.SECRET_KEY).decode("utf-8")+"~1", format="json")
+        token = jwt.encode(
+            {"email": self.user.email},
+            settings.SECRET_KEY).decode("utf-8")+"~1"
+        response = self.client.get(self.verify_url+"?token={}".format(token),
+                                   format="json")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
     def test_user_cant_activate_with_invalid_token(self):
         """Test should not activate an account with an invalid token."""
-        invalid_token = jwt.encode({"email": self.user.email, "exp": datetime.utcnow() -
-                                    timedelta(hours=23)}, settings.SECRET_KEY).decode("utf-8")+"1~"+str(self.user.id)
+        data = {
+            "email": self.user.email,
+            "exp": datetime.utcnow() - timedelta(hours=23)}
+        invalid_token = jwt.encode(data,
+                                   settings.SECRET_KEY)\
+            .decode("utf-8")+"1~"+str(self.user.id)
 
         response = self.client.get(
             self.verify_url+"?token="+invalid_token, format="json")
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        print('======', response.content)
```

Comment:
Please refer to the other comments about the print statements

Reply:
done bro